app/services/fees/calculator_test.go

```diff
@@ -0,0 +1,108 @@
+package fees
+
+import (
+	"github.com/limechain/hedera-eth-bridge-validator/config"
+	"github.com/limechain/hedera-eth-bridge-validator/test/mocks"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+const (
+	testConfigAddress = "../../../config/application.yml"
+	exchangeRate      = 0.0000764
+	validGasPrice     = "130"
+
+	invalidValue = "someinvalidvalue"
+
+	transferAmount        = "1000000000000"
+	invalidTransferAmount = "100"
+
+	transferFee         = "60000000000"
+	tooSmallTransferFee = "2"
+	tooBigTransferFee   = transferAmount
+)
+
+var (
+	hederaConfig = config.LoadConfigTest(testConfigAddress).Hedera
```

Comment:
Wouldn't it be better to mock the config instead. 
You are using only `configuration.Client.ServiceFeePercent` `configuration.Handler.ConsensusMessage.Addresses`, `BaseGasUsage` and `GasPerValidator`.

I don't think that uni tests should rely on env variables configuration